EssentialFeed/EssentialFeediOS/ImageComments UI/Controllers/ImageCommentCellController.swift

```diff
@@ -0,0 +1,34 @@
+
+import UIKit
+import EssentialFeed
+
+public final class ImageCommentCellController: NSObject {
+	private let viewModel: ImageCommentViewModel
+	private var cell: ImageCommentCell?
+
+	public init(viewModel: ImageCommentViewModel) {
+		self.viewModel = viewModel
+	}
+}
+
+extension ImageCommentCellController: UITableViewDataSource, UITableViewDelegate {
+	public func tableView(_ tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
+		1
+	}
+
+	public func tableView(_ tableView: UITableView, cellForRowAt indexPath: IndexPath) -> UITableViewCell {
+		cell = tableView.dequeueReusableCell()
+		cell?.labelUsername.text = viewModel.username
+		cell?.labelDate.text = viewModel.date
+		cell?.labelMessage.text = viewModel.message
+		return cell!
```

Comment:
Following my previous comment, we don't need to store a reference to the cell in a property:

```suggestion
		let cell: ImageCommentCell = tableView.dequeueReusableCell()
		cell.labelUsername.text = viewModel.username
		cell.labelDate.text = viewModel.date
		cell.labelMessage.text = viewModel.message
		return cell
```